backend/db/migrate/20250810025444_docuseal_removal.rb

```
@@ -0,0 +1,20 @@
+class DocusealRemoval < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)